### ISSUE TITLE
Add private message system

### ISSUE THREAD

**Issue by lzm11111111** [modrinth/code#2961]
### Please confirm the following.

- [x] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate feature requests
- [x] I have checked that this feature request is not on our [roadmap](https://roadmap.modrinth.com)

### What parts of Modrinth is your feature request related too?

Website

### Is your suggested feature related to a problem? Please describe.

Make peoples can send private messages to someone, this will make contact easier
I recommend that people only able to send private message once and only continue the chat after the other person responds

### Describe the solution you'd like

Ask for support

### Describe alternatives you've considered

May not

**Comment by Thinkseal** [comment]
I don't think that this is that's good of an idea (due to harassment, probably becoming a problem if this was implemented) and there is an alternatives for this that are already in modrinth, which are GitHub issues and discord server links. Both of which can be used to easily contact the creator about issues and suggestions
